target/scripts/startup/setup.d/mta_sts.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Set up MTA-STS
+
+function _setup_mta_sts() {
+  _log 'trace' 'Adding MTA-STS lookup to the Postfix TLS policy map'
```

Comment:
This is not "adding", but rather setting/overriding. If there is any existing config for this, it's replaced to only be for MTA-STS. We have a [helper for this that will avoid this concern](https://github.com/docker-mailserver/docker-mailserver/blob/d988d8a8d1be8bd07c271eaa9911f15f5cd033c7/target/scripts/helpers/postfix.sh#L102) 👍